src/ui/public/index_patterns/__tests__/_index_pattern.js

```diff
@@ -346,8 +346,7 @@ describe('index pattern', function () {
 
       it('is fulfilled by the result of interval toIndexList', async function () {
         const indexList = await indexPattern.toIndexList();
-        expect(indexList[0]).to.equal('foo');
-        expect(indexList[1]).to.equal('bar');
+        expect(indexList).to.equal('foo,bar');
```

Comment:
Does this worry anyone? I don't think this is a breaking change in the app itself but maybe we should return an array instead of a comma-delimited string?